concourse/tasks/e2e-test.yml

```diff
@@ -0,0 +1,26 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: gdscyber/concourse-chrome-driver
+    tag: latest
+params:
+  CGO_ENABLE: "0"
+  DEBIAN_FRONTEND: "noninteractive"
+  PYTHONIOENCODING: "UTF-8"
+inputs:
+- name: git-master
+outputs:
+- name: builds
+run:
+  path: /bin/bash
+  args:
+    - -euo
+    - pipefail
+    - -c
+    - |
+      echo "running e2e tests against staging..."
```

Comment:
Can we make it clear in pipeline output here that this is currently just a placeholder (as implementation of concourse_e2e in makefile currently does nothing).

Reply:
i have updated the make command to echo
`echo "TODO implement and run behave tests"`